packages/core/src/shared/plugins/suggestion/components/SuggestionList.tsx

```diff
@@ -59,13 +59,15 @@ export function SuggestionList<T extends SuggestionItem>(
 
   return (
     <div className={styles.menuList + " " + rootStyles.bnRoot}>
-      <MenuGroup>
-        {renderedGroups.length > 0 ? (
-          renderedGroups
-        ) : (
-          <Section title={"No match found"}> </Section>
-        )}
-      </MenuGroup>
+      <Menu defaultOpened={true} trigger={"hover"} closeDelay={10000000}>
```

Comment:
let's always make sure to comment workarounds like these (why is this necessary?), so we can easily understand it later :)